test/net/sourceforge/kolmafia/swingui/CalendarFrameTest.java

```diff
@@ -243,8 +244,8 @@ public void calendarFrameShouldHaveBasicFunctionality() {
 
   @Test
   public void itShouldCalculateExpectedCalendarDataForKnownTime() {
-    Calendar useTime = new GregorianCalendar();
-    useTime.set(2010, Calendar.SEPTEMBER, 1);
+    Calendar useTime = new GregorianCalendar(KoLmafia.KOL_TIME_ZONE);
```

Comment:
We have a GitHub or merge issue.      Calendar useTime = new GregorianCalendar(KoLmafia.KOL_TIME_ZONE); is already in main according to what my system thinks is main.

Reply:
Manually merged. I think I must have just based this off an older commit.